uberfire-nio2-backport/uberfire-nio2-impls/uberfire-nio2-jgit/src/main/java/org/uberfire/java/nio/fs/jgit/JGitFileSystemImpl.java

```diff
@@ -515,7 +517,19 @@ public boolean hasBeenInUse() {
     public void notifyExternalUpdate() {
         Object hook = fsHooks.get(FileSystemHooks.ExternalUpdate);
         if(hook != null){
-            JGitFSHooks.executeFSHooks(hook, FileSystemHooks.ExternalUpdate, name);
+            JGitFSHooks.executeFSHooks(hook, FileSystemHooks.ExternalUpdate, new FileSystemHookExecutionContext(name));
+        }
+    }
+
+    @Override
+    public void notifyPostCommit(int exitCode) {
```

Comment:
this methods is not covered by test :)

Reply:
This method will be covered by https://github.com/kiegroup/appformer/pull/520/commits/aa59a4d2209cc42fe82491073218ed10a37c43c0#diff-3b6c13f54617d3db33667cb964661e0d